frontend/src/lib/components/heroEmoji/HeroEmoji.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import type {HeroEmojiProps} from './HeroEmoji.type';
+
+  type $$Props = HeroEmojiProps;
+
+  // Merge the necessary classes to possible extra classes defined in the props
+  $$restProps.class = `text-center font-emoji text-[6.5rem] leading-[6.5rem] ${
+    $$restProps.class ?? ''
+  }`;
+</script>
+
+<!--
```

Comment:
are these comments used for some autogenerated documentation?

Reply:
Yes they are, see https://svelte.dev/docs/faq#how-do-i-document-my-components